protocol/migrations/sqlite/1719906191_add_community_token_version.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE community_tokens ADD COLUMN version TEXT DEFAULT "";
```

Comment:
Wdyt about `1.0` as default value?

Reply:
No, the default value needs to be empty, this mean that the contracts are old ones.
1.0.0 is the current version we have in the smart contract repo https://github.com/status-im/communities-contracts/blob/main/CHANGELOG.md#100-2024-02-28